drivers/dac/ad3552r/iio_ad3552r.c

```diff
@@ -82,7 +82,7 @@ struct iio_ad3552r_desc {
 /************************* Functions Definitions *****************************/
 /*****************************************************************************/
 
-static int iio_ad3552r_attr_get(void *device, char *buf, size_t len,
+static int iio_ad3552r_attr_get(void *device, char *buf, uint32_t len,
```

Comment:
both changes in this commit are good but i'd put this type change in a separate commit where i document in the description that the type was changed to match the struct iio_attribute show and store prototypes

Reply:
Thanks, done.